### DISCUSSION TITLE
Getting rpc error in gauge

### DISCUSSION THREAD

**Discussion by ayankumaar** [getgauge/gauge#2269] [General]
I am using gauge version 1.4.3 dotnet plugin 0.5.0. When I delete the bin and obj folder from my gauge script, I am getting the following error:

**rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp 127.0.0.1:55504: connectex: No connection could be made because the target machine actively refused it."**

Once I restart VS Code the error will be gone. But, if I again delete the bin and obj folder then the error will come back.

Is it a known issue in gauge ? If yes, then is there any work around for this ?

This is the setup that I have been using:

Gauge version: 1.4.3
Commit Hash: f98dd40

Plugins
-------
dotnet (0.5.0)
html-report (4.1.4)
screenshot (0.1.0)

Visual Studio Code: 1.67.2
.NET SDK: 6.0.301
OS: Windows 10

**Comment by vgoruslu** [comment]
I have the same problem. Is there any solution?
